tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,11 @@
 const App = () => {
-	return <div>A tic-tac-toe game</div>;
+	return (
+		<div>
+			<h1>A tic-tac-toe game</h1>
```

Comment:
Your components are set up in a modular way. This separation of concerns keeps your code maintainable and accessible for other developers. 👍